modules/loki-rbac-rules-permissions.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="loki-rbac-rules-permissions_{context}"]
+= Authorizing LokiStack rules RBAC permissions
+
+Administrators can allow users to create and manage their own alerting and recording rules by binding cluster roles to usernames.
+Cluster roles are defined as `ClusterRole` objects that contain necessary role-based access control (RBAC) permissions for users.
+
+In logging 5.8 and later, the following cluster roles for alerting and recording rules are available for LokiStack:
+
+[options="header"]
+|===
+|Rule name |Description
+
+|`alertingrules.loki.grafana.com-v1-admin`
+|Users with this role have administrative-level access to manage alerting rules across the entire cluster. This cluster role grants permissions to create, read, update, delete, list, and watch `AlertingRule` resources within the `loki.grafana.com/v1` API group.
```

Comment:
I think the "across the entire cluster" is still incorrect at this place, as this is decided when binding the role and not by the role itself. "admin" might be a bit misleading as a name, but it seems to be standardized. Essentially "admin" is just "read/write access", so the combination of "edit" and "view".

"crdview" is separate from the others as it concerns itself with the CustomResourceDefinition itself and not the resources and allows inspecting the definition. Regarding possible applications of this access permission I can think of two examples:

- operators checking for existence / content of the definition
- Editors (software on client-side) wanting to provide "auto complete" for the custom resources and thus needing to be able to access the definition

Reply:
ACK, sorry I missed this one!

Reply:
I think let's keep the scope to the roles for modifying the resources for now. The one for CRDs seems like a more advanced use case, and is sort of outside of the scope of what's actually being asked for in this Jira.